### PR TITLE
Calendar: describe selection properties

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/10 UI Components/dxCalendar/1 Configuration/disabledDates.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ Information about the checked date.
     href: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Calendar/Overview/"
 }
 
+In cases of *'multiple'* and *'range'* [selection modes](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectionMode), the behavior of disabled dates in Calendar is the following:
+
+- If you specify the [values](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#values) property programmatically, disabled dates in the values array are selected.
+
+- If you use UI to change selection (by clicks, Enter key, or [selecting a week on click](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectWeekOnClick)), you cannot select disabled dates in the *'multiple'* mode. In the *'range'* mode, disabled dates cannot be selected as the range start and range end, but can be included inside the range.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/disabledDates.md]: We usually don't use articles with mode names: in Edit mode, in Airplane mode, in Silent mode, etc. 

The rest is a few edits where I try to shorten the phrases a bit. 

```suggestion
- If you use UI to change selection (clicks on dates or [weeks](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectWeekOnClick), the Enter key), you cannot select disabled dates in *'multiple'* mode. In *'range'* mode, disabled dates cannot start or end a range, but can be included in the middle.
```

File: api-reference/10 UI Components/dxCalendar/1 Configuration/value.md [hunk 2]
```diff
@@ -5,26 +5,26 @@ default: null
 ---
 ---
 ##### shortDescription
-An object or a value specifying the date and time currently selected in the calendar.
+An object or a value that specifies the date and time currently selected in the calendar. Available only for *'single'* [selection mode](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectionMode).
 
 ---
-You can specify the current UI component value using any of the following formats.
+You can use the following formats to specify the current UI component value:
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/value.md]: "the current UI component value" - seems like a complicated phrase. I'm not sure if there's any specific reason you want to keep these words. But if you don't mind rewriting, I'd say something simple like this:

```suggestion
You can use the following date formats:
```

File: api-reference/10 UI Components/dxCalendar/1 Configuration/disabledDates.md [hunk 3]
```diff
@@ -20,6 +20,12 @@ Information about the checked date.
     href: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Calendar/Overview/"
 }
 
+In cases of *'multiple'* and *'range'* [selection modes](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectionMode), the behavior of disabled dates in Calendar is the following:
+
+- If you specify the [values](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#values) property programmatically, disabled dates in the values array are selected.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/disabledDates.md]:
```suggestion
- If you specify the [values](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#values) property programmatically, disabled dates are selected in the values array.
```

File: api-reference/10 UI Components/dxCalendar/1 Configuration/selectWeekOnClick.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether a user can select a week by a click on a week number.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/selectWeekOnClick.md]:
```suggestion
Specifies whether a user can select a week by clicking on a week number.
```

File: api-reference/10 UI Components/dxCalendar/1 Configuration/selectWeekOnClick.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ default: true
...
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+This property is in effect if [showWeekNumbers](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#showWeekNumbers) is enabled and [selectionMode](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectionMode) is either *'multiple'* or *'range'*.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/selectWeekOnClick.md]:
```suggestion
This property is in effect if [showWeekNumbers](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#showWeekNumbers) is enabled and [selectionMode](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectionMode) is *'multiple'* or *'range'*. 
```

File: api-reference/10 UI Components/dxCalendar/1 Configuration/selectionMode.md [hunk 6]
```diff
@@ -5,7 +5,16 @@ default: 'single'
...
+The following selection modes are available:
+
+- *'single'*    
+A user can select only one date at a time. The selected value is stored in the [value](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#value) property.
+
+- *'multiple'*     
+A user can select multiple dates at a time. The selected values are stored in the [values](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#values) property.
+
+- *'range'*      
+A user can select a range of dates. The first and the last range dates are stored in the [values](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#values) property.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/selectionMode.md]:
```suggestion
A user can select a range of dates. The first and the last date in the range are stored in the [values](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#values) property.
```

File: api-reference/10 UI Components/dxCalendar/1 Configuration/value.md [hunk 7]
```diff
@@ -5,26 +5,26 @@ default: null
 ---
 ---
 ##### shortDescription
-An object or a value specifying the date and time currently selected in the calendar.
+An object or a value that specifies the date and time currently selected in the calendar. Available only for *'single'* [selection mode](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectionMode).
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/value.md]:
```suggestion
An object or a value that specifies the date and time selected in the calendar. Available only for *'single'* [selection mode](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectionMode).
```

File: api-reference/10 UI Components/dxCalendar/1 Configuration/values.md [hunk 8]
```diff
@@ -5,7 +5,22 @@ default: []
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+An array of values that specify the date and time currently selected in the calendar. Available only for *'multiple'* and *'range'* [selection modes](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectionMode).
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxCalendar/1 Configuration/values.md]:
```suggestion
An array of values that specify the date and time selected in the calendar. Available only for *'multiple'* and *'range'* [selection modes](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#selectionMode).
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**